multiplayer_server/rooms/Game.php

```diff
@@ -427,12 +427,20 @@ public function remote_finish_race($player, $data)
         }
         $this->finish_race($player);
     }
+    
+    
+    private function ensure_time_format($time) {
+        if ($time < 0) {
+            $time = 0;
+        }
+        return sprintf("%.2f", $time);
```

Comment:
Since you don't actually _want_ a string, `return round($time, 2);` would be a better choice.

Reply:
What about typecasting to a float?

Reply:
It already is a float. What you want is to have two decimal places exactly, so you can't have .991 or .997 or any other problematic value. That's what `round()` does.

Reply:
Gotcha.  If I don't add this first, @jacob-grahn will.